tests/functional/test_locked_objects.py

```diff
@@ -543,12 +552,12 @@ def _cluster_with_components(cluster: Cluster, hosts: List[Host]):
     return first_service_component, second_service_component
 
 
-def _lock_obj(obj: Union[Cluster, Service, Component, Provider, Host]) -> Task:
+def _lock_obj(obj: Union[Cluster, Service, Component, Provider, Host], lock_action: str = "lock") -> Task:
```

Comment:
There is `AnyADCMObject` in `tests/functional/plugin_utils.py`, maybe move it to general functional utils and use it in typing like here

Reply:
Fixed